test/org/sagebionetworks/bridge/models/surveys/TestSurvey.java

```diff
@@ -167,6 +167,7 @@ public TestSurvey(boolean makeNew) {
         setCreatedOn(DateUtils.getCurrentMillisFromEpoch());
         setVersion(2L);
         setPublished(true);
+        setSchemaRevision(42);
```

Comment:
Is schemaRevision editable through the API? It looks like it could be changed and submitted through the UI. Do you want to prevent this?